src/tasks/test/BackgroundKeyTest.py

```
@@ -0,0 +1,107 @@
+        self.description = "验证后台模式下按键能否送达游戏（置顶+等待+按键+恢复），输出诊断日志"
+        self.visible = self.debug
+
+        self.default_config = {
+            "后台模式启用": True,  # 强制后台模式，直接测试后台按键路径
+            "测试按键": "m",
+            "测试次数": 3,
+            "按键后等待(秒)": 1.0,
+            "仅测试置顶": False,
+        }
+        self.config_description = {
+            "测试按键": "要测试的按键（默认 m 打开地图，结果可观察）",
```

Ruff (0.16.2)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)